addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8488,7 +8488,7 @@ msgid "Reset PVR database"
 msgstr ""
 
 msgctxt "#19186"
-msgid "All data in the PVR database is being erased. Are you sure?"
+msgid "All data in the PVR database will be erased. Are you sure?"
 msgstr ""
```

Comment:
as we have a TV and EPG database and not a PVR database we should change it to be more generic.

"All your Live TV related data (channels, groups, guide) will be cleared. Are you sure?"

and for id 19185

"Clear database" is enough IMO because we are in the settings context for Live TV